recipe/meta.yaml

```diff
@@ -43,16 +58,41 @@ requirements:
 test:
   imports:
     - unstructured_client
-  commands:
-    - pip check
+  source_files:
+    - gh_src/_test_unstructured_client
+    - gh_src/_sample_docs/*
+    - gh_src/README.md
   requires:
     - pip
+    - pytest
+    - pytest-asyncio
+    - pytest-mock
+    # requests-mock not available for py312
+    - requests-mock  # [py<312]
+    - requests-toolbelt
+    # not used by the unit tests
+    # - pypdf
+    # - deepdiff
+    # - uvloop
+  commands:
+    - pip check
+    - cd gh_src/
+    # requests-mock not available for py312
+    - pytest _test_unstructured_client/unit -v  # [py<312]
+    - pytest _test_unstructured_client/unit/test_split_pdf_hook.py -v  # [py>=312]
 
 about:
-  summary: Python Client SDK for Unstructured API
   home: https://github.com/Unstructured-IO/unstructured-python-client
   license: MIT
   license_file: LICENSE.md
+  license_family: MIT
+  summary: Python Client SDK for Unstructured API
+  description: |
+    Unstructured provides a platform and tools to ingest and process
+    unstructured documents for Retrieval Augmented Generation (RAG)
+    and model fine-tuning.
+  doc_url: https://docs.unstructured.io/welcome
```

Comment:
Would this welcome page be the `home` and the `doc_url` be the docs link in the `README`,`https://docs.unstructured.io/api-reference/api-services/sdk-python`?

Reply:
updated, thanks for catching this.

I set 

```
  home: https://docs.unstructured.io/api-reference/api-services/overview
...
  doc_url: https://docs.unstructured.io/api-reference/api-services/sdk-python
```